### DISCUSSION TITLE
Mac电脑 蓝灯付费用户 一直现实连接US 但是google/油管都打不开

### DISCUSSION THREAD

**Discussion by zisforever** [getlantern/lantern-forum-cn#1039] [General]
如题，这样子的情况已经很久了，更新到了最新的7.9.5版本也不行？麻烦回复指导一下。

**Comment by MaxMKMax** [comment]
@zisforever 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@zisforever 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。技术部门想了解您具体是遇到什么问题, 这样才能帮您解决问题.

**Reply by zisforever** [reply]
好的，我马上试一下，然后在程序里提交log报告

**Reply by zisforever** [reply]
我已经在程序里提交了问题report, 辛苦查看解决一下
